src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java

```diff
@@ -534,7 +534,6 @@ public OffenderBookingIdSeq getOffenderIdentifiers(final String offenderNo, fina
         return offenderIdentifier;
     }
 
-    @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER"})
```

Comment:
Not required as this is now only called from 2 places which both have the VerifyBookingAccess annotation